media/js/src/utils.jsx

```diff
@@ -58,4 +58,152 @@ export const formulaText = (content = { title: '', body: [] }, i) => (
                 tex={eq} />
         ))}
     </li>
-);
\ No newline at end of file
+);
+
+export const dataAttr = {
```

Comment:
I think we should take this out of utils and make it its's own data attributes file.